react/week4/user_search/src/Login.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState, useContext } from "react";
+import { UserContext } from "./UserContext";
+
+export default function Login({ setUserId }) {
+	const { userId } = useContext(UserContext);
```

Comment:
```suggestion
	const { userId, setUserId } = useContext(UserContext);
```
To solve an error: Cannot update a component (`App`) while rendering a different component (`Login`).